src/components/stepforms/RollupStepForm.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div>
+    <StepFormHeader :title="title" :stepName="this.editedStep.name" />
+    <MultiselectWidget
+      id="hierarchyColumnsInput"
+      v-model="editedStep.hierarchy"
+      name="Hierarchal columns (from lowest to highest level):"
```

Comment:
```suggestion
      name="Hierarchical columns (from lowest to highest level):"
```
Also a change here if you agree to reverse the order